src/sfdxCommand.ts

```diff
@@ -145,7 +145,7 @@ export abstract class SfdxCommand extends Command {
   protected flags!: OutputFlags<any>; // eslint-disable-line @typescript-eslint/no-explicit-any
 
   // The parsed args for easy reference by this command; assigned in init
-  protected args!: OutputArgs<any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+  protected args!: OutputArgs;
```

Comment:
this was not used by CLI (and from looking in oclif I don't see that it worked previously).  In any case, I can be persuaded that it's a breaking change.

Reply:
I'd vote against this change. I think we need to be as delicate as possible with this change. Anyone else using `SfdxCommand` could be affected by it... we saw what happened when messing with types in core

Reply:
the underlying oclif/command doesn't have `OutputArgs` as a generic.  That's what prompted the change.  Sorry for omitting that important explanation

Reply:
@mdonnalley changed it.